lib/mayaUsd/ufe/UsdUndoRenameCommand.cpp

```diff
@@ -60,6 +60,14 @@ UsdUndoRenameCommand::UsdUndoRenameCommand(const UsdSceneItem::Ptr& srcItem, con
 {
     const UsdPrim& prim = _stage->GetPrimAtPath(_ufeSrcItem->prim().GetPath());
 
+    // eraly check to see if a particular node has any specs to contribute
```

Comment:
"eraly" --> "Early"

Reply:
> 
> 
> Looks good, as far as I can tell. I would have liked to see a test demonstrating the impact of this fix, i.e. before your fix the test would fail, and after your fix it would pass.

That's me being lazy!  Please see 7fafef624444489ef3ddddd200706b9a57ed353c

Fyi, The sample usda file comes from Christopher in this issue ticket:
https://github.com/Autodesk/maya-usd/issues/330